{
	"id": "CVE-2023-38470",
	"vulnerabilities": [
		{
			"id": "CVE-2023-38470",
			"description": "A vulnerability was found in Avahi. A reachable assertion exists in the avahi_escape_label() function.",
			"severity": [
				{
					"type": "cvss_v31",
					"source": "security.alpinelinux.org",
					"cvss_v31": {
						"vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
						"base_score": 5.5,
						"base_severity": "MEDIUM",
						"temporal_score": 5.5,
						"temporal_severity": "MEDIUM",
						"environmental_score": 5.5,
						"environmental_severity": "MEDIUM"
					}
				}
			],
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2023-38470"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2191690"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://access.redhat.com/security/cve/CVE-2023-38470"
				}
			],
			"published": "2023-11-02T15:15:08Z",
			"modified": "2024-03-13T14:08:39.049654Z"
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:3.19",
			"vulnerable": true,
			"package": {
				"name": "avahi"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "0.8-r15"
					}
				],
				"fixed": [
					"0.8-r15"
				]
			}
		}
	],
	"data_source": "alpine-osv"
}
